src/pagination/pagination.component.ts

```diff
@@ -41,60 +42,85 @@ import { BehaviorSubject } from "rxjs";
 		[ngClass]="{
 			'bx--skeleton' : skeleton
 		}">
-
+		<!-- left skeleton div -->
 		<div *ngIf="skeleton" class="bx--pagination__left">
 			<p class="bx--skeleton__text" style="width: 70px"></p>
 			<p class="bx--skeleton__text" style="width: 35px"></p>
 			<p class="bx--skeleton__text" style="width: 105px"></p>
 		</div>
 
 		<div *ngIf="!skeleton" class="bx--pagination__left">
-			<span class="bx--pagination__text">{{itemsPerPageText | async}}</span>
-			<div class="bx--form-item">
-				<div class="bx--select bx--select--inline">
-					<label
-						[for]="itemsPerPageSelectId"
-						class="bx--label bx--visually-hidden">
-						{{itemsPerPageText | async}}
-					</label>
-					<select
-						[id]="itemsPerPageSelectId"
-						[(ngModel)]="itemsPerPage"
-						class="bx--select-input"
-						aria-describedby="false">
-						<option class="bx--select-option" value="10">10</option>
-						<option class="bx--select-option" value="20">20</option>
-						<option class="bx--select-option" value="30">30</option>
-						<option class="bx--select-option" value="40">40</option>
-						<option class="bx--select-option" value="50">50</option>
-					</select>
-					<svg
-						class="bx--select__arrow"
-						fill-rule="evenodd"
-						height="5"
-						role="img"
-						viewBox="0 0 10 5"
-						width="10"
-						[attr.aria-label]="optionsListText | async"
-						[attr.alt]="optionsListText | async">
-						<title>{{optionsListText | async}}</title>
-						<path d="M0 0l5 4.998L10 0z"></path>
-					</svg>
-				</div>
+			<!-- <span class="bx--pagination__text">{{itemsPerPageText | async}}</span> -->
+			<div class="bx--select bx--select--inline">
+				<label
+					[for]="itemsPerPageSelectId"
+					class="bx--label bx--visually-hidden">
+					{{itemsPerPageText | async}}
+				</label>
+				<select
+					[id]="itemsPerPageSelectId"
+					[(ngModel)]="itemsPerPage"
+					class="bx--select-input"
+					aria-describedby="false">
+					<option class="bx--select-option" value="10">{{itemsPerPageText | async}} 10</option>
```

Comment:
Carbon components experimental demo has the number first, and no colon. We can probably ignore colon for now but open an issue to change it before X.
```suggestion
					<option class="bx--select-option" value="10">10 {{itemsPerPageText | async}}</option>
```

Reply:
Also, pagination story doesn't look right in standard any more.